src/ess/livedata/handlers/to_nxlog.py

```diff
@@ -59,19 +66,16 @@ def _ensure_capacity(self, data) -> None:
             )
 
     def add(self, timestamp: int, data: LogData) -> None:
-        self._ensure_capacity(data.value)
+        self._ensure_capacity(data)
         self._timeseries.coords['time'].values[self._end] = data.time
         self._timeseries.data.values[self._end] = data.value
+        if data.variances is not None and self._timeseries.data.variances is not None:
+            self._timeseries.data.variances[self._end] = data.variances
```

Comment:
Is it a problem that we silently ignore variances of `data` here if the timeseries has no variances? Or that we silently do not add variances if `data` has none but the timeseries does?

Reply:
Highly unlikely, since `self._timeseries` was made from previous calls to this method, i.e., the "data" are just new additions to the same thing, based on a single stream of messages. The double-check is actually pointless but satisfies linters.